lib/rules/selector-not-notation/index.mjs

```diff
@@ -199,6 +225,41 @@ function fixComplex(previousNot) {
 		node = node.next();
 		prev.remove();
 	}
+
+	ruleNode.selector = root.toString();
+
+	return getNewSource(previousNot, ruleNode);
+}
+
+/**
+ * @param {Pseudo} pseudo
+ * @param {RuleNode} ruleNode
+ */
+function getSource(pseudo, ruleNode) {
+	assert(ruleNode.source?.start && pseudo.source?.start && pseudo.source?.end);
+	const startline = ruleNode.source.start.line + pseudo.source.start.line - 1;
+	const endline = ruleNode.source.start.line + pseudo.source.end.line - 1;
+
+	return {
+		start: { line: startline, column: pseudo.source.start.column },
+		end: { line: endline, column: pseudo.source.end.column + 1 },
+	};
+}
+
+/**
+ * @param {Pseudo} pseudo
+ * @param {RuleNode} ruleNode
+ * @param {number} [offset]
+ */
+function getNewSource(pseudo, ruleNode, offset = 0) {
+	assert(ruleNode.source?.start && pseudo.source?.start);
+	const line = ruleNode.source.start.line + pseudo.source.start.line - 1;
+	const column = pseudo.source.start.column;
+
+	return {
+		start: { line, column },
+		end: { line, column: column + pseudo.toString().length + offset },
```

Comment:
WIP

This one doesn't return the right column.
I gotta choose between not returning it—which is not necessary—or fixing it.